apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html

```diff
@@ -501,6 +578,69 @@ <h1>{{log.operationType}}
 
             <div class="address-row-wrapper d-flex">
                 <div class="address-row mt-05">
+                    <!-- old account address start -->
+                    <ng-container *ngIf="log?.old && log?.old?.details && log?.old?.details?.addresses && log?.old?.details?.addresses.length">
```

Comment:
HTML repetition can be avoided with:
`<div class="single-address-row" *ngIf="index===0 || index>0 && showSingleAddress && isShowMultipleDataIndex===parentIndex">`